playbooks/viaq-store.yml

```diff
@@ -113,16 +113,8 @@
         - "openshift-ansible-lookup-plugins"
         - "openshift-ansible-playbooks"
         - "openshift-ansible-roles"
-    - name: Copy inventory and vars.yml
```

Comment:
this wont work as its being installed on fresh CentOS/RHEL and both inv and vars are not present

Reply:
I generate and copy the inventory and vars.yml with the namespaces. So the files should already be on the metrics store machine.